.github/workflows/manual-docker-build.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Manual - Docker Build
```

Comment:
`Manual - Docker Build` --> `Manual - Docker Build and Test`

Reply:
updated the workflow name